tagstudio/src/qt/widgets/tag_box.py

```diff
@@ -50,6 +63,13 @@ def __init__(
         self.base_layout.setContentsMargins(0, 0, 0, 0)
         self.setLayout(self.base_layout)
 
+        self.tag_entry = QLineEdit()
```

Comment:
it looks weird when the input field is jumping left and right depending on how does it momentarily fit next to the existing tags. Maybe it would be better to keep the input field on a separate line, so it's always aligned on the left? Up to you.

Reply:
If I can figure out how, I will.